app/helpers/comments_helper.rb

```diff
@@ -291,4 +291,37 @@ def fallback_url_for_comment(comment, options = {})
 
   end
 
+  # find the parent of the commentable
```

Comment:
@elzj Yep! Otherwise you can end up with it trying to get the ultimate parent of the chapter or work.